man/systemd.network.xml

```diff
@@ -992,6 +992,26 @@
             <para>Specifies the outgoing device to match. The outgoing interface is only available for packets originating from local sockets that are bound to a device.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>SourcePort=</varname></term>
+          <listitem>
+            <para>Specifies the source port or port range match in forwarding information base (FIB) rules.
+            A port range is specified by the lower and upper port separated by a dash. Defaults to unset.</para>
```

Comment:
Could you update the explanation about 'unset'. Maybe all port?

Reply:
sure unset means  0 . I see in kernel if the code ```struct fib_rule *nlrule ``` is defaults to allocated as kzalloc